python/python/bystro/prs/preprocess_for_prs.py

```diff
@@ -332,7 +347,10 @@ def finalize_dosage_scores_after_c_t(
     genos_adjusted = merged_allele_comparison_genos.apply(adjust_dosages, axis=1)
     genotypes_adjusted_only = genos_adjusted.iloc[:, 1:]
     genos_transpose = genotypes_adjusted_only.T
-    return genos_transpose, scores_after_c_t
+    with tempfile.NamedTemporaryFile(delete=False, suffix=".feather") as tmpfile:
```

Comment:
I'm a little concerned that we are reading through the dosage matrix 3 times by this point. Also, what is the memory saving if `dosage_overlap` is already in memory? Can't we just return genos_transpose directly, instead of writing to temp storage and reading back again in the caller `generate_c_and_t_prs_scores`